hetznerbot/helper/hetzner.py

```diff
@@ -297,10 +307,15 @@ def format_offers(subscriber, offer_subscriber, get_all=False):
         price = offer.price / 100
         price_incl_vat = float(offer.price) * 1.19 / 100
 
+        # Get cpu data
+        cpu = session.get(Cpu, offer.cpu)
+
         # First chunk of data
         updated_date = offer.last_update.strftime("%d.%m - %H:%M")
         formatted_offer = f"""*Offer {offer.id} {offer_status}:* [ {updated_date} ]
-_Cpu:_ {offer.cpu}
+_Cpu:_ {cpu.name} ({cpu.release})
+- *{cpu.cores}* cores, *{cpu.threads}* threads
+- Multi: *{cpu.multi_thread_rating}*, Single: *{cpu.single_thread_rating}*
```

Comment:
The info is now getting a bit crowded.
I think adding 2/4 spaces in front of these helps a lot regarding readability.

You should probably add it to the HDD list while at it.

Reply:
done